calico_kubernetes/calico_kubernetes.py

```diff
@@ -1,23 +1,27 @@
 #!/bin/python
-import json
+from __future__ import print_function
```

Comment:
nope its used once. print_function turns `print "statement"` into `print("statement")`.
The print_function can be unittested but __builtin__.print can't be easily tested